docs/examples/index.rst

```diff
@@ -6,6 +6,7 @@ Here are several end-to-end examples of using Gluon NLP Toolkit for modeling tas
    :maxdepth: 1
 
    word_embedding/word_embedding.ipynb
-   word_embedding_evaluation/word_embedding_evaluation.ipynb
+   word_embedding/word_embedding_toolkit.ipynb
+   word_embedding/word_embedding_evaluation.ipynb
```

Comment:
seems that the content of embedding evaluation notebook already exists in the toolkit notebook. did you intend to delete evaluation notebook?